SECURITY_CONTACTS

```diff
@@ -0,0 +1,15 @@
+# Reporting a security vulnerability
+
+Please do:
+  - not disclose any security issue publicly e.g. Pull Requests, Comments.
```

Comment:
...security issue(s) publicly e.g. through GitHub Pull Requests, Issues and/or Comments.

Reply:
I understand that the sentence could be more explicit but I would like to advocate for not changing it as it does not add that much benefit IMO for the following reasons.

-in case of "security issue"  IMO if the statement is prohibiting in relation to a single entity then it is prohibiting in relation to multiples of the same entity in this context. So I don't think we need the (s) at the end .

- related to the second adjustment, I think if we ask the contributor to not disclose the issue publicly then we also prohibit the contributor on all platform. I don't think it is important to highlight that it is a GitHub PR or a GitLab PR as both of them are undesired.
 
I trust the contributors will understand what we mean by this statement, if they don't we can always change it in the future.